core/src/validation.jl

```diff
@@ -441,6 +441,39 @@ function valid_demand(
     return !errors
 end
 
+function valid_outlet_crest_level(graph::MetaGraph, outlet::Outlet, basin::Basin)::Bool
+    errors = false
+    for (id, crest) in zip(outlet.node_id,outlet.min_crest_level)
+        id_in = inflow_id(graph, id)
+        if id_in.type == NodeType.Basin
+            #validation:
+            basin_bottom_level = basin_bottom(basin, id_in)
+            if crest == -Inf
+                crest == basin_bottom_level[2]
```

Comment:
`crest == basin_bottom_level[2]` checks if they are equal, perhaps you meant `=`? Note though that that would not change the value in the `outlet.min_crest_level`, only `outlet.min_crest_level[i] = ...` would do that.

Reply:
Ops, it should be `=`